src/main/java/Application.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class Application {
+    public static void main(String[] args) {
+        Scanner scanner =  new Scanner(System.in);
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분");
+        List<String> racingCarName = Arrays.stream(scanner.nextLine().split(","))
+                .map(String::strip)
+                .collect(Collectors.toList());
```

Comment:
toList 메소드를 사용할 수 있습니다!
```suggestion
                .toList();
```

Reply:
확인했습니다!